addons/storysource/src/StoryPanel.js

```diff
@@ -5,6 +5,8 @@ import { Link } from '@storybook/router';
 import { SyntaxHighlighter } from '@storybook/components';
 
 import { createElement } from 'react-syntax-highlighter';
+import prism from 'react-syntax-highlighter/dist/styles/prism/prism';
```

Comment:
I don't think that you should use dist in the import path

Reply:
However you maybe have no choice

Reply:
Got the example code from readme. 
Don't see any other options. https://github.com/conorhastings/react-syntax-highlighter/blob/master/README.md#prism

Reply:
ok :+1:

Reply:
This might be a more versatile approach to styling the code-editor then the theming api, but I'm not quite happy with how this would bundle ALL prism themes into the manager, even if the user isn't even using any.

Can we make this lazy loading somehow?

Reply:
Why would even make that default ? I mean it looks ok to pass the style from the outside, and the default could be set in `config.js`.